.travis.yml

```diff
@@ -3,24 +3,26 @@ cache: packages
 sudo: true
 dist: trusty
 
-addons:
-  apt:
-    packages:
-      - libfreetype6
-
-r:
-- 3.1
-- 3.2
-- oldrel
-- release
-- devel
+# build matrix; skip vdiffr on all but r release
+matrix:
+  include:
+  - r: 3.1
+    env: SKIP_VDIFFR=true
```

Comment:
Can we flip this around to make skipping the default, so we only set `VDIFFR=true` for R-release?

Reply:
Oh that makes sense. Maybe we could turn off by default when the TRAVIS (IIRC) env var is ?

Reply:
I find having a separate variable we define slightly preferable, because that means vdiffr runs can be fully controlled in any testing environment where these tests may be run, whether on travis or not.

Reply:
I was thinking we could do both - if `USE_VDIFFR` is set, use its value, otherwise look at `TRAVIS` and don't run vdiffr if set.

Reply:
Ok, this seems to be working now. I put the broken image back in as a check to see which builds run vdiffr.